ext/standard/formatted_print.c

```diff
@@ -644,7 +644,16 @@ php_formatted_print(zend_execute_data *execute_data, int use_array, int format_o
 			inpos++;
 		}
 	}
-
+	
+	if (argc - 1 > argnum) {
+		efree(result);
+		if (newargs) {
+			efree(newargs);
+		}
+		php_error_docref(NULL, E_WARNING, "Too much arguments");
```

Comment:
Not sure if we should keep this message consistent with the existing warning for fewer arguments than expected.

Reply:
Not related to functionality, but `too many` instead of `too much` is more grammatically correct :grin:

Reply:
Otherwise yes, in such case the more descriptive warning helps better when debugging code.

Reply:
@petk I always confused when to use `much` and `many`. In this case, indeed, `many` should be used as `arguments` are something countable :sweet_smile: